lib/seccomp-tools/asm/compiler.rb

```diff
@@ -129,10 +139,17 @@ def compile_cmp(op, val, jt, jf)
       end
 
       def label_offset(label)
+        if label.is_a?(Integer)
+          raise ArgumentError, "Loop detected!" if label == 0
+          raise ArgumentError, "Does not support backward jumping." if label < 0
+          return label
+        end
         return label if label.is_a?(Integer)
```

Comment:
Remove this line

Reply:
Don't forget this

Reply:
fyi, check was a little bit superfluous since supplying a negative number would error anyways:
```
Error: Expected a number or label name, while "-13124" occured
```